### ISSUE TITLE
Request : Support more than 2 persons in the transcript  and audio

### ISSUE THREAD

**Issue by pfloresl** [souzatharsis/podcastfy#130]
I imagine this is not supported, but it would be great if we could enable more than two roles in a conversation about the uploaded content.

**Comment by souzatharsis** [comment]
This is interesting. Could you please elaborate why this is important for
your use case?

<http://linkedin.com/in/tharsissouza>

**Comment by pfloresl** [comment]
Yes, a mechanism for learning is to, for example, have philosophers, artists, and scientists have a conversation about a topic, each bringing up their background. Each will show to the others something that they may not be seeing. 
Many times I listen to programs that involve 2-5 people, each representing their world.

This provides a rich conversation

**Comment by souzatharsis** [comment]
Thanks; that's helpful.

We should generalize the number of speakers.
<http://linkedin.com/in/tharsissouza>

**Comment by brumar** [comment]
Hi @souzatharsis 
I thought a bit about this and I think the dev branch supports this (but the current configuration files won't work, nor the llm backend that was set up emulate the current default transcript engine) .

**Comment by brumar** [comment]
Sure, please use a MP when you need my attention, I have many podcastfy notifications and I could miss it.
